src/core.rs

```diff
@@ -1046,6 +1100,16 @@ impl Core {
         }
     }
 
+    fn retry_bootstrap(&mut self) {
+        self.crust_service.stop_bootstrap();
+        self.state = State::Disconnected;
+        for &connection in self.proxy_map.keys() {
+            self.crust_service.drop_node(connection);
+        }
+        self.proxy_map.clear();
+        self.crust_service.bootstrap(0u32, Some(CRUST_DEFAULT_BEACON_PORT));
```

Comment:
I agree; I'll change that, run `ci_test` and add it to the PR.
(It won't change anything for `ci_test`, though, as the nodes currently can only try to connect to the beacon, I think.)

@brian-js Are you fine with that?